ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/PendingTransactions.java

```diff
@@ -96,7 +96,7 @@ public boolean addRemoteTransaction(final Transaction transaction) {
     return addTransaction;
   }
 
-  boolean addLocalTransaction(final Transaction transaction) {
+  public boolean addLocalTransaction(final Transaction transaction) {
```

Comment:
You should move all of the `TransactionPool` related classes to `eth`.  I believe that includes: `TransactionPool`, `PendingTransactions`, `PendingTransactionListener`, `PendingTransactionDroppedListener`.

Reply:
Yeah -- I was initially wondering about whether or not to do that -- I've updated the branch accordingly